config-windows.md

```diff
@@ -0,0 +1,3 @@
+# Windows Application Container Configuration
+
+Windows application containers can be configured using the following properties.
```

Comment:
On Thu, Sep 01, 2016 at 04:11:39PM -0700, Rob Dolin (MSFT) wrote:
> > @@ -0,0 +1,3 @@
> > +# Windows Application Container Configuration
> > +
> > +Windows application containers can be configured using the following properties.
>
> … My desire with this PR it clear that it is the intent of OCI to
> support Linux, Solaris, and Windows.

Then how about dropping this file (until you have Windows-specific
settings to put into it) and using:

  * Windows containers: [runtime.md](runtime.md) and [config.md](config.md)

in README.md?